identity/src/test/java/com/stripe/android/identity/navigation/DocSelectionFragmentTest.kt

```diff
@@ -248,8 +187,9 @@ internal class DocSelectionFragmentTest {
                     )
                 )
                 setUpSuccessVerificationPage()
-                binding.singleSelectionContinue.findViewById<MaterialButton>(R.id.button)
-                    .callOnClick()
+                // trigger navigation
+//                binding.singleSelectionContinue.findViewById<MaterialButton>(R.id.button)
+//                    .callOnClick()
```

Comment:
should this still be commented out?

Reply:
Yes, this test is ignored, and this line won't compile because we replaced binding with compose. But Just want to keep it there to remind adding navigation trigger when later the test is reenabled.